### PR TITLE
updates to group

### PR DESCRIPTION
**Pull request by davidmu1**

### CODE REVIEW

File: api-reference/beta/resources/user.md [hunk 1]
```diff
@@ -168,6 +168,7 @@ This resource supports:
 | id | String | The unique identifier for the user. Inherited from [directoryObject](directoryobject.md). <br><br>Returned by default. Not nullable. Read-only.|
 | identities | [objectIdentity](objectIdentity.md) collection | Represents the identities that can be used to sign in to this user account. An identity can be provided by Microsoft (also known as a local account), by organizations, or by social identity providers such as Facebook, Google, and Microsoft, and tied to a user account. May contain multiple items with the same **signInType** value. <br><br>Returned only on $select. Supports $filter. |
 | imAddresses | String collection | The instant message voice over IP (VOIP) session initiation protocol (SIP) addresses for the user. Read-only.|
+| infoCatalogs | String collection | Identifies the info segments assigned to the group. Returned by default. |
```
Comment (krbain) [inline api-reference/beta/resources/user.md:171]: This should say "assigned to the user" instead of group.

File: concepts/changelog.md [hunk 2]
```diff
@@ -33,9 +33,11 @@ For a summary of the value of these API changes, as well as recent tools, compon
...
+| Addition | beta | Added the **onPremisesPublishing** property to [application](/graph/api/resources/application?view=graph-rest-beta) entity. |
 | Addition | beta | Added the **OnPremisesPublishing** property to [Update Application](/graph/api/application-update?view=graph-rest-beta). |
 | Addition | beta | Added new resources [authenticationFlowsPolicy](/graph/api/resources/authenticationflowspolicy?view=graph-rest-beta) and [selfServiceSignUpAuthenticationFlowConfiguration](/graph/api/resources/selfservicesignupauthenticationflowconfiguration?view=graph-rest-beta) and [get](/graph/api/authenticationflowspolicy-get?view=graph-rest-beta) and [update](/graph/api/authenticationflowspolicy-update?view=graph-rest-beta) operations.|
+| Addition | v1.0 | Added the **assignedLabels**, **expirationDateTime**, **membershipRule**, **membershipRuleProcessingState**, **preferredLanguage**, and **theme** properties to the [group](/graph/api/resources/group?view=graph-rest-v1.0) entity. |
+| Addition | beta | Added the **infoCatalogs** property to the [group](/graph/api/resources/group?view=graph-rest-v1.0) entity and the [user](/graph/api/resources/user?view=graph-rest-beta) entity. |
```
Comment (krbain) [inline concepts/changelog.md:40]: Shouldn't the change also be included in v1.0 for users?

File: api-reference/beta/resources/user.md (hunk 1 above)
Comment (yyuank) [reply api-reference/beta/resources/user.md]: Replaced "group" with "user"

File: concepts/changelog.md (hunk 2 above)
Comment (yyuank) [reply concepts/changelog.md]: Corrected the changelog. infoCatalog is added to user entity on beta, not v1.

**Pull request closed without merging**